ngboost/tests/test_distns.py

```diff
@@ -36,7 +40,7 @@ def product_list(*its: Iterable) -> List:
 @pytest.mark.parametrize(
     ["dist", "score", "learner"],
     product_list(
-        [Normal, LogNormal, Exponential],
+        [Normal, LogNormal, Exponential, T, TFixedDf, TFixedDfFixedVar, Cauchy],
         [LogScore, CRPScore],
```

Comment:
specifically here. I guess we can move these into a seperate test, or break down the parametrisation to be more granular. By the way, I have to say, the `product_list` here seems kinda unnecessary if we can just do three `pytest.mark.parametrize` decorators chained after each other

Reply:
I agree on both points here, I think a separate test here would be simple enough but I'm okay with either route. Is it noted in the docs somewhere that CRPScore will not work with these?

Reply:
Not sure about the docs specifically, but it's in the docstring of the classes @angeldroth wrote i think

Reply:
cool yeah just thinking of the inevitable issues :)

Reply:
I've split the tests up and fixed an error with TFixedDfLogScore